csrc/sys_utils.h

```diff
@@ -34,4 +34,7 @@ class LibraryLoader : public NonCopyable {
   void* handle_ = nullptr;
 };
 
+// Return true if compute-sanitizer is attached
+NVF_API bool detectComputeSanitizer();
```

Comment:
```suggestion
bool detectComputeSanitizer();
```

https://github.com/NVIDIA/Fuser/blob/main/doc/dev/visibility.md#should-i-mark-my-new-method-or-class-as-nvf_api

Reply:
Oh, I thought it's necessary to make it visible to the tests.